test/errhandling/psahabu/catch-parens.chpl

```diff
@@ -0,0 +1,19 @@
+use ThrowError;
+
+try {
+  throwAnError();
+} catch () {
```

Comment:
This case seems pretty weird to me. I'm not sure why anybody, even a C/C++ programmer, would write that...